holding_verification.py

```diff
@@ -24,16 +29,18 @@ def main():
     table_name = default_config["CHECKSUM_TABLE_NAME"]
 
     db_function = sqlite3.connect(db_file_name)
-    app_core = HoldingVerificationCore(db_function, table_name)
+    enter = yellow("Enter")
+    csv_file_name_prefix = input(
+        f"Add a title to be prepended to the CSV result's file name then '{enter}' or just press '{enter}' to skip: "
+    ).replace(" ", "_")
```

Comment:
should this do a trim before replace, that way any accidental spaces at the end of string or just an empty blank string will not cause unnecessary underscores

Reply:
Oh yes! Good point; I'll add that